sass.py

```diff
@@ -584,6 +596,21 @@ def _get_file_arg(key):
             'not {1!r}'.format(SassFunction, custom_functions)
         )
 
+    _custom_exts = kwargs.pop('custom_import_extensions', []) or []
+    if isinstance(_custom_exts, (text_type, bytes)):
+        _custom_exts = [_custom_exts]
```

Comment:
hmm, not sure how I feel about a parameter which ends with `s` that can take a `str`, let's instead raise a `TypeError` and require an iterable of some sort

Also let's just require that they are `text` or ascii compatible:

```python
custom_import_extensions = [
    ext.encode('UTF-8')
    for ext in kwargs.pop('custom_import_extensions', ())
]
```